backend/hitas/views/apartment.py

```diff
@@ -676,11 +703,21 @@ class Meta:
 
 class ApartmentListSerializer(ApartmentDetailSerializer):
     type = serializers.SerializerMethodField()
+    has_conditions_of_sale = serializers.SerializerMethodField()
+    has_grace_period = serializers.SerializerMethodField()
 
     @staticmethod
     def get_type(instance: Apartment) -> Optional[str]:
         return getattr(getattr(instance, "apartment_type", None), "value", None)
 
+    @staticmethod
+    def get_has_conditions_of_sale(instance: Apartment) -> bool:
+        return instance.has_conditions_of_sale  # Annotated field
```

Comment:
You can annotate this method with `hitas.models.apartment.ApartmentWithAnnotations` and add the `has_conditions_of_sale` there.